### PR TITLE
Functions "apiVersion", "name" and "icon" added; read all CIP-30 wallet names from windows.cardano

### PR DESCRIPTION
**Pull request by dduehr**
Hi Anton,
thanks for your Cardano CIP30 Purescript interface. I added three functions of the basic CIP30 API. Just startet to experiment with Purescript and Cardano Web3.
Kind regards, Dirk

### CODE REVIEW

File: src/Cardano/Wallet.purs [hunk 1]
```diff
@@ -59,9 +64,18 @@ type Cip30DataSignature =
 -- | Wrapper for wallet names. see nami, flint, etc for examples.
 newtype WalletName = WalletName String
 
+apiVersion :: WalletName -> Effect String
```
Comment (anton-k) [inline src/Cardano/Wallet.purs]: Use `getApiVersion`

File: src/Cardano/Wallet.purs [hunk 2]
```diff
@@ -59,9 +64,18 @@ type Cip30DataSignature =
...
+apiVersion walletName = _apiVersion walletName
+
+name :: WalletName -> Effect String
```
Comment (anton-k) [inline src/Cardano/Wallet.purs]: Use `getName`

File: src/Cardano/Wallet.purs [hunk 3]
```diff
@@ -59,9 +64,18 @@ type Cip30DataSignature =
...
+name walletName = _name walletName
+
+icon :: WalletName -> Effect DataUri
```
Comment (anton-k) [inline src/Cardano/Wallet.purs]: Use `getIcon`

**Review by anton-k — CHANGES_REQUESTED** [review]

File: src/Cardano/Wallet.purs (hunk 1 above)
Comment (dduehr) [reply src/Cardano/Wallet.purs]: Yep, renamed it. The naming convention was quite obvious actually. But I didn't see it nonetheless.

File: src/Cardano/Wallet.purs (hunk 2 above)
Comment (dduehr) [reply src/Cardano/Wallet.purs]: Renamed.

File: src/Cardano/Wallet.purs (hunk 3 above)
Comment (dduehr) [reply src/Cardano/Wallet.purs]: Renamed.

**Review by anton-k — APPROVED** [review]

**Comment by anton-k** [comment]
Thank you for PR!

**Pull request merged**